rpcd/etc/openstack_deploy/user_variables.yml

```diff
@@ -62,3 +62,9 @@ ceph_mons: >
   {% endfor -%}
   {% endif -%}
   {{ _var }}
+
+galera_pip_packages:
```

Comment:
It would be helpful to have a comment here referencing the issue tracking the eventual fix.